src/get_examples.py

```diff
@@ -3,10 +3,18 @@
 import glob
 import json
 import os
+from pathlib import Path
 
 
 ROOT = os.path.realpath(os.path.join(os.path.dirname(__file__), ".."))
 EXAMPLES = os.path.join(ROOT, "examples")
+IGNORE_CHANGES = [
```

Comment:
These get incorporated into the docs, but only if the `build_docs` is done. Is this the case?

Reply:
I don't know what you mean exactly. What I did here is to avoid that changes to these files trigger CI builds of all the examples, if that answers the question 😅

Reply:
I will add a comment to the variable explaining how it is used